db/seeds/users.rb

```diff
@@ -1,188 +1,188 @@
+puts User.create(first: 'Plum', last: 'Chocolate', email: 'PlumChocolate@Chocolate.com', user_name: 'PlumChocolate', street: '6722 Main St.' , city: 'Ashbury', state: 'AR', zip: 27482, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Souffle', last: 'Chupa', email: 'SouffleChupa@Chupa.com', user_name: 'SouffleChupa', street: '87906 Main St.' , city: 'Paris', state: 'TX', zip: 44444, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Topping', last: 'Chupa', email: 'ToppingChupa@Chupa.com', user_name: 'ToppingChupa', street: '7499 Main St.' , city: 'Boston', state: 'MA', zip: 37182, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Powder', last: 'Chupa', email: 'PowderChupa@Chupa.com', user_name: 'PowderChupa', street: '3234 Main St.' , city: 'Denver', state: 'CO', zip: 80202, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Jelly', last: 'Chups', email: 'JellyChups@Chups.com', user_name: 'JellyChups', street: '47722 Main St.' , city: 'Springfield', state: 'MO', zip: 57294, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Beans', last: 'Chups', email: 'BeansChups@Chups.com', user_name: 'BeansChups', street: '38808 Main St.' , city: 'Miami', state: 'FL', zip: 66666, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Beans', last: 'Claw', email: 'BeansClaw@Claw.com', user_name: 'BeansClaw', street: '97847 Main St.' , city: 'Boston', state: 'MA', zip: 37182, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Souffle', last: 'Cookie', email: 'SouffleCookie@Cookie.com', user_name: 'SouffleCookie', street: '70575 Main St.' , city: 'Miami', state: 'FL', zip: 66666, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Croissant', last: 'Cookie', email: 'CroissantCookie@Cookie.com', user_name: 'CroissantCookie', street: '85245 Main St.' , city: 'Topeka', state: 'KS', zip: 94625, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Pie', last: 'Cotton', email: 'PieCotton@Cotton.com', user_name: 'PieCotton', street: '71036 Main St.' , city: 'Fairfax', state: 'VA', zip: 22033, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Sugar', last: 'Cotton', email: 'SugarCotton@Cotton.com', user_name: 'SugarCotton', street: '96502 Main St.' , city: 'Topeka', state: 'KS', zip: 94625, password: 'password', password_confirmation: 'password', admin: false)
+puts User.create(first: 'Plum', last: 'Cream', email: 'PlumCream@Cream.com', user_name: 'PlumCream', street: '77005 Main St.' , city: 'Newtown', state: 'CT', zip: 164
```

Comment:
Line is too long. [241/80]<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.